CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Karafka Sidekiq Backend
 
-## Unreleased
+## 1.4.4 (Unreleased)
 - Ruby 3.1 support
-- Drop support for ruby 2.6
+- Drop support for ruby 2.6 in the CI
```

Comment:
We require ruby >= 2.7 in the gemspec, we should change this back to `Drop support for ruby 2.6`